src/test/java/core/basesyntax/service/ReportCreatorServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service;
+
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.ReportCreatorServiceImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class ReportCreatorServiceTest {
+
+    @Test
+    public void reportCreatorService_createList_OK() {
+        final Map<String, Integer> testMap = new HashMap<>(3);
+        testMap.put("banana", 30);
+        testMap.put("apple", 50);
+        testMap.put("pinapple", 22);
+        ReportCreatorService reportCreatorService = new ReportCreatorServiceImpl();
+        List<String> testList = reportCreatorService.createReport(testMap);
+        assertFalse(testList.isEmpty());
+    }
+
+    @Test
+    public void reportCreatorService_ListValuesMatch_OK() {
```

Comment:
the naming of test methods should be like this:
![Screenshot 2022-07-08 at 09 37 26](https://user-images.githubusercontent.com/60258373/177931842-ee60521c-aced-4d51-9755-2ee1034d41be.png)
fix naming for all test methods

Reply:
Yes, You are right.